### PR TITLE
Pl/pandas

### PR DESCRIPTION
**Pull request by pl88**

### CODE REVIEW

File: features/data_handler_class.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import os
+import pandas as pd
+import pandas_datareader.data as pd_dr        
+
+class DataHandlerClass():
+    
+    def __init__(self) -> None:
+        data = pd.DataFrame
+        stock = ''
+        path_to_file = ''
```
Comment (angon1) [inline features/data_handler_class.py]: Consider:
```python
    def __init__(self) -> None:
        self._data = pd.DataFrame
        self._stock = ''
        self._path_to_file = ''
```
'self' -> Refers to object itself, so 'self._variable' reffers to to variable assigned to the instance of class

File: main.py [hunk 2]
```diff
@@ -1,28 +1,56 @@
...
+    wig20.append(Stocks('Mbank', 'mbk'))
+    wig20.append(Stocks('Orange pl', 'opl'))
+    wig20.append(Stocks('PEKAO', 'peo'))
+    wig20.append(Stocks('Pepco', 'pco'))
+    wig20.append(Stocks('PGE', 'pge'))
+    wig20.append(Stocks('PGNiG', 'pgn'))    
+    wig20.append(Stocks('PKNOrlen', 'pkn'))
+    wig20.append(Stocks('PKO BP', 'pko'))
+    wig20.append(Stocks('PZU', 'pzu'))
+    wig20.append(Stocks('Santander pl', 'spl'))
```
Comment (angon1) [inline main.py]: Please consider:
```python
input_data = [
            ('Allegro', 'ale'),
            ('Asseco Poland', 'acp'),
            (...),
            ]
wig20 = []
for item in input_data:
     wig20.append(Stocks(item))
```

File: main.py [hunk 3]
```diff
@@ -1,28 +1,56 @@
...
+from features.plot_data_class import PlotDataClass
 
+class Stocks():
+        def __init__(self, _name, _ticker) -> None:
+            self.name = _name
+            self.ticker = _ticker
+            self.dataHandler = DataHandlerClass()
+            self.dataHandler.download(self.ticker)
+            StockDataParserClass.calculate_daily_change(self.dataHandler.data)
+            self.dataHandler.store_data_to_csv()
```
Comment (angon1) [inline main.py]: Are you sure that you want to save csv in Stocks() __init__?

**Review by angon1 — APPROVED** [review]

File: features/data_handler_class.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+from ntpath import join
```
Comment (angon1) [inline features/data_handler_class.py]: Are you sure this is needed?
`ntpath.join` is not `str.join`. I think you are using str.join in your code.

Comment (pl88) [reply features/data_handler_class.py]: True, need to disable auto import in VS Code...

**Pull request merged**